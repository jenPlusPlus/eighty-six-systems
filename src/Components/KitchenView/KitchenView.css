.table-manager-button {
  border: none;
  font-size: 12px;
  margin: 5px 0 5px 0;
  width: 90px;
  height: 30px;
  border-radius: 15px;
  background-color: #EFEFF0;
  color: #6D576A; }

.kitchen-view-order {
  background-color: #33202A;
  position: relative;
  width: 60%;
  margin: 10px auto 10px auto;
  border-radius: 15px;
  padding: 5px; }

.remove-from-kitchen-view {
  border: none;
  font-size: 12px;
  margin: 5px 0 5px 0;
  width: 90px;
  height: 30px;
  border-radius: 15px;
  background-color: #EFEFF0;
  color: #6D576A;
  position: absolute;
  top: 5px;
  right: 5px; }

.orders-wrapper {
  height: 80%; }

.all-orders {
  height: 87%;
  overflow-y: scroll; }

.order-table-info {
  position: absolute;
  top: 5px;
  left: 5px; }

.seat-order-card {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: space-around;
  margin-top: 45px; }

.seat-order {
  background-color: #EFEFF0;
  color: #33202A;
  border-radius: 10px;
  margin: 5px;
  padding: 5px;
  width: 45%;
  font-weight: bolder; }

.seat-order-item {
  color: #33202A;
  margin-top: 3px;
  font-weight: normal; }
