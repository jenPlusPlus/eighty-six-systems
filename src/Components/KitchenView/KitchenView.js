import React, { Component } from 'react';

class KitchenView extends Component {
  constructor() {
    super();
  }

  render() {
    return (
      <div>KITCHENVIEW!</div>
    );
  }
}

export default KitchenView;
