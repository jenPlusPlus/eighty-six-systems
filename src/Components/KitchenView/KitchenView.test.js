import { shallow } from 'enzyme';
import KitchenView from './KitchenView';
import ReactDOM from 'react-dom';
import React from 'react';

describe('KitchenView snapshot', () => {

  it('should always match the snapshot', () => {
    const wrKitchenViewer = shallow(<KitchenView
      allOrders={[]}
      currentUser={{name: 'Jen', loginCode: '0000'}}/>);

    expect(wrKitchenViewer).toMatchSnapshot();
  });
});
