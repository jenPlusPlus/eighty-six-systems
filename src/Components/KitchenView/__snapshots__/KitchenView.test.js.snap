// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KitchenView snapshot should always match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <KitchenView
    allOrders={Array []}
    currentUser={
        Object {
            "loginCode": "0000",
            "name": "Jen",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Link
          replace={false}
          to="/0000/tables"
>
          <button>
                    Table Manager
          </button>
</Link>,
        <h2>
          All Orders
</h2>,
        <div
          className="all-orders"
>
          
</div>,
      ],
      "className": "orders-wrapper",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <button>
            Table Manager
</button>,
          "replace": false,
          "to": "/0000/tables",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Table Manager",
          },
          "ref": null,
          "rendered": "Table Manager",
          "type": "button",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "All Orders",
        },
        "ref": null,
        "rendered": "All Orders",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
          "className": "all-orders",
        },
        "ref": null,
        "rendered": Array [],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Link
            replace={false}
            to="/0000/tables"
>
            <button>
                        Table Manager
            </button>
</Link>,
          <h2>
            All Orders
</h2>,
          <div
            className="all-orders"
>
            
</div>,
        ],
        "className": "orders-wrapper",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <button>
              Table Manager
</button>,
            "replace": false,
            "to": "/0000/tables",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Table Manager",
            },
            "ref": null,
            "rendered": "Table Manager",
            "type": "button",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "All Orders",
          },
          "ref": null,
          "rendered": "All Orders",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [],
            "className": "all-orders",
          },
          "ref": null,
          "rendered": Array [],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
